aws/data_source_aws_instance_test.go

```diff
@@ -601,6 +643,13 @@ resource "aws_instance" "test" {
     device_name  = "/dev/sde"
     virtual_name = "ephemeral0"
   }
+
+  ebs_block_device {
+    device_name = "/device/sdf"
```

Comment:
See [documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html).

```suggestion
    device_name = "/dev/sdf"
```

Reply:
Whoops, I think that may have been something with my autocomplete